### ISSUE TITLE
[question] Running event file on tensorboard

### ISSUE THREAD

**Issue by shreyshrivastava4799** [RustingSword/tensorboard_logger#1]
Hi there, I was able to generate the output event file in demo folder named as tfevents.pb. My question is how do I visualise this on tensorboard as you have depicted in your README. I tried running `tensorboard --logdir=demo` which I verified using  `find demo | grep tfevents` is able to find the events file but opening the link in the web browser doesn't show any output. 
Thanks.

Also, it would be really helpful if you could mention the protobuf version as 3.11 somewhere [this link](https://github.com/protocolbuffers/protobuf/blob/master/src/README.md) helped install properly. And in Makefile had to use `LDFLAGS =  -pthread -I/usr/local/include -L/usr/local/lib -lprotobuf ` for compiling ( output of `pkg-config --cflags --libs protobuf` ).  Thanks for helping. Brilliant work !!

**Comment by RustingSword** [comment]
Hi, I tested it again just now in Windows WSL (Ubuntu 18.04, with gcc 7.4.0, protobuf 3.0.0), and didn't see any issues. Is your tensorboard working properly with other tfrecord files? I attached my generated file if you want to verify. 
[tfevents.zip](https://github.com/RustingSword/tensorboard_logger/files/4243571/tfevents.zip)

As for the protobuf version, I think as long as it supports proto3 it's OK. That being said, PR is welcome!
